client/apps/edit/components/layout/index.coffee

```diff
@@ -116,7 +116,7 @@ module.exports = class EditLayout extends Backbone.View
 
   getBodyText: =>
     @fullText = []
-    if @article.get('lead_paragraph').length
+    if @article.get('lead_paragraph')?.length
```

Comment:
I wonder if on the server-side we should be stricter—either not allow nulls or not allow blank strings. Generally speaking on think I got a little `allow('', null)` heavy [in the schema](https://github.com/artsy/positron/blob/master/api/apps/articles/model/schema.coffee#L93) and we could be more strict about validation server-side to avoid the consumer apps from having to be extra defensive. e.g. In this case `allow(null).min(3)` would be a more appropriate way to describe "An article can have no lead paragraph, or a lead paragraph at-least 3 characters long". Not suggesting for this PR, but wanted to put that out as food for thought going forward.